packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -70,6 +70,16 @@ object DatadogSDKWrapperStorage {
 }
 
 internal class DatadogSDKWrapper : DatadogWrapper {
+    internal var rumForLogsState = RUM_ENABLE_LOGS_DEFAULT
+    internal var traceForLogsState = TRACES_ENABLE_LOGS_DEFAULT
```

Comment:
if those two are only for the integration of Logs with RUM/Traces, maybe it is worth moving them to `DdLogsImplementation`?

I would also suggest renaming to indicated that it is boolean (generally speaking state means more than 2 states, imo), like `bundleWithTraces: Boolean` / `bundleWithRum: Boolean`, this will also make API a bit more consistent with naming.

Reply:
I'll rename them, but regarding moving them - I'm defining them this way to enable the user to put them in the configuration. If they were directly inside DdLogsImplementation, how would the user override the values? The sdk does not communicate directly with the logs class, and the logs implementation class doesn't read from the configuration

Reply:
okay, let's keep them here